docs/components/receive.md

```diff
@@ -305,6 +313,8 @@ Flags:
                                  Path to YAML file that contains object
                                  store configuration. See format details:
                                  https://thanos.io/tip/thanos/storage.md/#configuration
+      --receive.async-workers=5  Number of concurrent workers processing
+                                 incoming remote-write requests.
```

Comment:
Are they processing incoming remote writes or forwarding the replication requests? These are two very different things.

Reply:
My suggestion is to rename this to `receive.forward.async-workers` or something similar.